recipes/icecream-cpp/all/conanfile.py

```diff
@@ -0,0 +1,15 @@
+from conans import ConanFile, tools
+
+class IcecreamcppConan(ConanFile):
+    name = "icecream-cpp"
+    license = "MIT"
+    url = "https://github.com/renatoGarcia/icecream-cpp"
+    description = "A little library to help with the print debugging on C++11 and forward."
+    topics = ("debug", "single-header-lib", "print")
+    no_copy_source = True
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+
+    def package(self):
+        self.copy(f'icecream-cpp-{self.version}/icecream.hpp', 'include/', keep_path=False)
```

Comment:
No f-strings (yet).
Older python versions are still supported.

Reply:
Ok. Fixed that.